ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -204,6 +204,15 @@ Verify Pipeline Run Is Completed
     Wait Until Page Contains Element    //span[@class='pf-v5-c-label__text' and text()='Succeeded']    timeout=${timeout}
     Capture Page Screenshot
 
+# robocop: disable:line-too-long
+Verify Experiment Has Completed
+    [Documentation]    Check that the "Last 5 runs" field of an "Experiments and runs" screen reports success
+    [Arguments]    ${experiment_name}    ${pipeline_run_name}    ${timeout}=10m
```

Comment:
I think ${pipeline_run_name} is not being used

Reply:
True, I'd remove the parameter if I still wanted to use the keyword.